### ISSUE TITLE
Rule proposal: `prefer-modern-math-apis`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1527]
I'm not good at math, but I found people use the complicated one, maybe because IE don't support it.

## Fail

```js
const foo = Math.log(bar) * Math.LOG10E;
```

```js
const foo = Math.LOG10E * Math.log(bar);
```

```js
const foo = Math.log(bar) / Math.LN10;
```

## Pass

```js
const foo = Math.log10(bar);
```

`Math.log2` seems the same, so maybe we should come up a better name.

**Comment by sindresorhus** [comment]
`prefer-modern-math-apis`? (Like `prefer-modern-dom-apis`)

**Comment by sindresorhus** [comment]
This is now accepted.
